src/WebJobs.Extensions.SendGrid/Bindings/SendGridScriptBindingProvider.cs

```diff
@@ -57,7 +57,7 @@ public override bool TryResolveAssembly(string assemblyName, out Assembly assemb
 
             if (string.Compare(assemblyName, "SendGridMail", StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
To avoid issues like this, I went with a different approach with the Twilio implementation:
https://github.com/Azure/azure-webjobs-sdk-extensions/blob/master/src/WebJobs.Extensions.Twilio/Bindings/TwilioScriptBindingProvider.cs#L26
Might be worth doing the same here.